src/browser/browser.js

```diff
@@ -37,6 +37,7 @@ define((require, exports, module) => {
   const Model = Record({
     version: '0.0.7',
     mode: 'create-web-view', // or show-web-view, edit-web-view, choose-web-view
+    transition: 'zoom', // or fade, none
```

Comment:
Maybe it's better to use `null` instead of  `'none'` ? I don't feel strongly on either option, just a thought I had when I saw `'none'`.